include/mbedtls/x509_crt.h

```diff
@@ -560,6 +559,20 @@ typedef int (*mbedtls_x509_crt_ca_cb_t)( void *p_ctx,
                                          mbedtls_x509_crt const *child,
                                          mbedtls_x509_crt **candidate_cas );
 
+/*
+ * \brief       Sort the server cert chain to handle potentially extraneous
+ *              certificates and arbitrary orderings.
+ *              For more detail pelease check:
+ *           https://tools.ietf.org/id/draft-ietf-tls-tls13-23.html#certificate.
+ *
+ * \note        Performs the same job as \c mbedtls_crt_verify_with_profile()
```

Comment:
This seems to be a copy-paste mistake from the documentation of `mbedtls_crt_verify_xxx()`. Could you please remove it?

Reply:
fixed.